packages/flutter/lib/src/cupertino/route.dart

```diff
@@ -709,3 +714,72 @@ class _CupertinoEdgeShadowPainter extends BoxPainter {
     canvas.drawRect(rect, paint);
   }
 }
+
+/// Displays an iOS-style dialog above the current contents of the app.
+///
```

Comment:
Describe a bit why this function specifically exists (e.g. that it does the right iOS mimicking animations etc)